### ISSUE TITLE
DBN for Univariate series with a dicision label

### ISSUE THREAD

**Issue by PradyumnaSripad** [albertbup/deep-belief-network#30]
Greetings sir,

My code for univariate classification for 5 classes is

X=df['value'].values;y = df['class']
dbn = UnsupervisedDBN(hidden_layers_structure=[1,5],
                      batch_size=10,
                      learning_rate_rbm=0.06,
                      n_epochs_rbm=20,
                      activation_function='sigmoid')

The error i got is 
Please help me to rectify the same!
![issue](https://user-images.githubusercontent.com/32504327/39911793-373fcc7c-551a-11e8-80b2-523beb67260d.PNG)

**Comment by albertbup** [comment]
Hi PradyumnaSripad,

I've solved the bug in PR #31 
Re-install the package and try again, it should work now.

Best.
